source/macos/AdaptiveCards/AdaptiveCards/Views/ACRCollectionViewItem.swift

```diff
@@ -2,37 +2,41 @@ import AdaptiveCards_bridge
 import Cocoa
 
 class ACRCollectionViewItem: NSCollectionViewItem {
-    static let identifier = NSUserInterfaceItemIdentifier(rawValue: "MyItem")
+    static let identifier = NSUserInterfaceItemIdentifier(rawValue: "ImageSetItem")
     
     override func loadView() {
-        view = itemImageView
+        view = itemView
     }
     
-    private let itemImageView: NSImageView = {
-        let view = NSImageView()
+    private let itemView: NSView = {
+        let view = NSView()
         view.translatesAutoresizingMaskIntoConstraints = false
-        view.imageAlignment = .alignCenter
         return view
     }()
```

Comment:
If I'm not wrong these lines shouldn't be required anymore.

Reply:
removed